pkg/config/containers.conf

```diff
@@ -121,6 +121,16 @@ default_sysctls = [
 #
 #env_host = false
 
+#Set the ip for the host.containers.internal entry in the containers /etc/hosts
```

Comment:
The comments should be prefixed with a whitespace to make it easier to brain-parse.

Reply:
Yes of course, copy paste mistake.